pkg/diagnostics/diagnostics.go

```diff
@@ -67,6 +68,17 @@ func CmdAnalyze() *cobra.Command {
 	return cmd
 }
 
+// CmdReport returns a CLI command
+func CmdReport() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "report",
+		Short: "Reports the proxy status",
```

Comment:
The command will not only run a report of the proxy, but it will be extended to other issues as well.
```suggestion
		Short: "Run reports of the status and setup",
```

Reply:
updated, Thanks